frontend/src/components/TaskCreator/GroupMemberPicker.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactElement, KeyboardEvent } from 'react';
+import { connect } from 'react-redux';
+import { NONE_TAG, NONE_TAG_ID } from 'common/lib/util/tag-util';
+import { State, Tag } from 'common/lib/types/store-types';
+import TagListPicker from '../Util/TagListPicker/TagListPicker';
+import styles from './Picker.module.css';
+import SamwiseIcon from '../UI/SamwiseIcon';
+
+type OwnProps = {
+  readonly tag: string;
+  readonly opened: boolean;
+  readonly onTagChange: (tag: string) => void;
+  readonly onPickerOpened: () => void;
+};
+type Props = OwnProps & {
+  // subscribed from redux store.
+  readonly getTag: (id: string) => Tag;
+};
+
+function GroupMemberPicker({
+  tag,
+  opened,
+  onTagChange,
+  onPickerOpened,
+  getTag,
+}: Props): ReactElement {
+  // Controllers
+  const clickPicker = (): void => { onPickerOpened(); };
+  const pressedPicker = (e: KeyboardEvent): void => {
+    if (e.key === 'Enter' || e.key === ' ') onPickerOpened();
+  };
+  const reset = (): void => onTagChange(NONE_TAG_ID);
+  // Nodes
+  const displayedNode = (isDefault: boolean): ReactElement => {
+    const { name, color, classId } = getTag(tag);
+    const style = isDefault ? { background: NONE_TAG.color } : { background: color };
+    const internal = isDefault
+      ? (
+        <>
+          <span className={styles.TagDisplay}>
+            <SamwiseIcon iconName="user-plus" className={styles.CenterIcon} tabIndex={-1} />
+            {' '}
+            assign to&nbsp;&nbsp;
+          </span>
+        </>
+      )
+      : (
+        <>
+          <span className={styles.TagDisplay}>{classId != null ? name.split(':')[0] : name}</span>
+          <button type="button" className={styles.ResetButton} onClick={reset}>&times;</button>
+        </>
+      );
+    return (
+      <span
+        role="presentation"
+        onClick={clickPicker}
+        onKeyPress={pressedPicker}
+        className={styles.Label}
+        style={style}
+      >
+        {internal}
+      </span>
```

Comment:
is this supposed to be a button?

Reply:
:octocat-astonished:

Reply:
This is the current placeholder for the GroupMemberPicker, but this code itself is identical to the one existing in TagPicker. I can fix this in each of the `_`Pickers in the next commit.

Reply:
Upon further inspection, I just tried to fix this, first by replacing this `<span>` tags with `<button type="button">`, but it turned out to not only be a couple lines fix and was more involved (styles behave differently for both tags, and there are now default styles that must be overridden/changed). I will note this and create a new issue for later fixing (as it already preexists in the current DatePicker and TagPicker), but it does not appear to be vital to me in our current timeline.